other-builds/experimental/audio-echo/app/build.gradle

```diff
@@ -0,0 +1,56 @@
+apply plugin: 'com.android.model.application'
+
+// pointing to cmake's source code for the same project
+def REMOTE_PROJ_ROOT = '../../../../' + rootProject.getName() + '/' +
+        project.getName() + '/src/main'
+
+model {
+    android {
+        compileSdkVersion = 23
+        buildToolsVersion = '23.0.2'
+
+        defaultConfig {
+            applicationId='com.google.sample.echo'
+            minSdkVersion.apiLevel = 17
+            targetSdkVersion.apiLevel = 23
+            versionCode = 1
+            versionName = '1.0'
+        }
+        ndk {
+            platformVersion = 23
```

Comment:
Oh, this is just to show that we do have a control to configure native code build [and it could be configured to different values]